babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-react'],
+  env: {
+    test: {
+      presets: ['@babel/preset-env', '@babel/preset-react'],
+      plugins: ['@babel/plugin-transform-modules-commonjs'],
```

Comment:
Would be good to install `@babel/plugin-transform-modules-commonjs` in the `package.json` too.